lib/active_model/serializer/utils.rb

```diff
@@ -23,7 +42,9 @@ def include_args_to_hash(included)
     when Symbol
       { included => {} }
     when Hash
-      included.each_with_object({}) { |(key, value), hash| hash[key] = include_args_to_hash(value) }
```

Comment:
Why the multiline? If you think it's better as a multiline, it should be a `do`...`end` block.